html-css/week1/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400..700;1,400..700&family=Press+Start+2P&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css" crossorigin="anonymous">
+    <title>Elnaz Talebi portfolio web page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="nav-container">
+    <nav class="navbar">
+      <ul class="nav-list">
+        <li>
+          <a href="#welcome-section">About</a>
+        </li>
+        <li>
+          <a href="#projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#contacts">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    </div>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Hey, I am Elnaz</h1>
+      <br>
```

Comment:
In this case, I would recommend using margin to add space between `h1 `and `p` tags. 

The `<br> `element produces a line break in text (carriage-return). It is useful for writing a poem or an address, where the division of lines is significant.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br